code/modules/client/preferences.dm

```diff
@@ -43,6 +43,7 @@ datum/preferences
 	var/backbag = 2						//backpack type
 	var/backbag_style = 1
 	var/pda_choice = 2
+	var/headset_choice = 2
```

Comment:
```suggestion
	var/headset_choice = OUTFIT_HEADSET
```

And below.

Or is there a reason nothing is using their defines, here?

Reply:
originally i was gonna undef them